internal/alertmanager/currentalert/process.go

```diff
@@ -91,11 +94,17 @@ func (e *examiner) processAlert(ds disableScaling) (bool, error) {
 
 	switch e.Alert.Annotations["command"] {
 	case "addPVC":
+		if err := checkLabelExists(e.Alert.Labels, "persistentvolumeclaim"); err != nil {
```

Comment:
I'd introduce types for alerts and define a validation function which could be invoked after filtering.